docs/source/overview.rst

```diff
@@ -2,37 +2,29 @@
 Overview
 ========
 
-What does it currently do
+Neo-python is an implementation of the Neo protocol using Python. 
+
+
+What neo-python does
 ^^^^^^^^^^^^^^^^^^^^^^^^^
 
-- This project aims to be a full port of the original C# `NEO project <https://github.com/neo-project>`_
-- Run a Python based P2P node
-- Interactive CLI for configuring node and inspecting blockchain
-- Compile, test, deploy and run Smart Contracts written in python or any Smart Contract in the ``.avm`` format
-- `NEP2 <https://github.com/neo-project/proposals/blob/master/nep-2.mediawiki>`_ and `NEP5 <https://github.com/neo-project/proposals/blob/master/nep-5.mediawiki>`_ compliant wallet functionality
-- RPC Client
-- RPC server
-- Notification Server ( for viewing transfers of NEP5 tokens )
-- ``Runtime.Log`` and ``Runtime.Notify`` event monitoring
-
-What will it do
-^^^^^^^^^^^^^^^
+- Full node implementation compatible with `Neo blockchain <https://github.com/neo-project/neo/tree/master-2.x>`_.
+- Command Line Interface (CLI) for node configuration and blockchain inspection.
+- Smart Contracts development using `neo-boa <https://github.com/CityOfZion/neo-boa>`_, debugging and deployment using the command line.
+- `RPC Client <https://github.com/CityOfZion/neo-python-rpc>`_ and RPC Server.
+- Notifications plugins with optional RPC endpoint.
+- ``Runtime.Log`` and ``Runtime.Notify`` event monitoring.
 
-- Consensus nodes
-- Full smart contract debugging and inspection
 
 
 Getting started
 ^^^^^^^^^^^^^^^
-Please follow directions in `the install section <install.html>`_
+Please follow directions in `the install section <install.html>`_.
 
-The main functionality for this project is contained within the cli application ``np-prompt``.  You can `view usage details here <prompt.html>`_
+Use ``np-prompt`` to start the application.  You can `view usage details here <prompt.html>`_.
```

Comment:
Please keep the original line. Reasoning:

"The main functionality for this project" suggests that there is more than just `np-prompt`. Which is true, because we have `np-api-server` which is participating in the network without a CLI, we also have standalone tools for importing and exporting raw blocks such that you do not need to be connected to the P2P network layer at all and can just process blocks into a database that you can work with (e.g. do statistics).